workflows/cloudify_system_workflows/dsl_import_resolver/resolver_with_catalog_support.py

```diff
@@ -230,8 +231,8 @@ def _upload_missing_plugin(self, name, specifier_set, distribution):
     @staticmethod
     def matching_distro_wagon(wagons, distro):
         matching_wagons = [x['url'] for x in wagons
-                           if x['name'].lower() == distro
-                           or x['name'].lower().startswith('manylinux')]
+                           if x['release'].lower() == distro
+                           or x['release'] == 'manylinux']
```

Comment:
surely we can't require manylinux exactly? manylinux1 and manylinux2010 exist and are technically different from manylinux